src/v/config/configuration.cc

```diff
@@ -1124,18 +1124,18 @@ configuration::configuration()
       "Max age of metadata cached in the health monitor of non controller node",
       {.needs_restart = needs_restart::no, .visibility = visibility::tunable},
       10s)
-  , storage_space_alert_free_threshold_percent(
+  , storage_alert_free_threshold_percent(
       *this,
-      "storage_space_alert_free_threshold_percent",
+      "storage_alert_free_threshold_percent",
```

Comment:
will that change influence backward compatibility i.e. if a value was set in cluster configuration will it be applied to the property with changed name ?

Reply:
Read up on our policy on configuration compatibility.. Chances are I will remove this part (and will need to rebase and fix the conflicts).

Reply:
I've removed this change for now. I'll leave the rename as a nice-to-have future TODO.